build.gradle

```diff
@@ -52,14 +52,14 @@ dependencies {
 	// hard dependency -> actually includes it in our jar
 	include "net.kyrptonaught:customportalapi:${project.portal_api_verion}"
 	include("maven.modrinth:Wd844r7Q:${project.structure_api_verion}")
+	include "curse.maven:DisableCustomWorldsAdvice-401978:${project.dcwa_file}"
```

Comment:
Only doing this is not enough. This just includes in in our jar, but doesn't link it.

Testing in game shows it doesn't work in dev environment. Might need modRuntimeOnly or modLocalRuntime or something to use it.

Reply:
Tested with `modLocalRuntime` on both dev and packages jar and it seems to work.